thunder/tests/test_grad.py

```diff
@@ -1640,3 +1640,21 @@ def foo(a, c):
     static_trace = thunder.last_backward_traces(static_jit)[-1]
     # static trace should bake `c` as scalar number, so it won't show up in backward as proxy
     assert not any(map(lambda x: isinstance(x, Proxy), tree_flatten(static_trace.args[0])[0]))
+
+
+def test_get_saved_for_backward_tensors():
+    from thunder.core.vjp_utils import get_saved_for_backward_tensors
+
+    def func(a, b):
+        return a * b
+
+    a = make_tensor((2, 2), device="cpu", dtype=torch.float32, requires_grad=True)
+    b = make_tensor((2, 2), device="cpu", dtype=torch.float32, requires_grad=True)
```

Comment:
Probably better to also test when the tensors do not require grad

Reply:
Done.